internal/pkg/extensions/validate.go

```diff
@@ -72,7 +72,16 @@ func (ext *Extension) validateContents() error {
 
 		// no symlinks
 		if d.Type().Type() == os.ModeSymlink {
-			return fmt.Errorf("symlinks are not allowed: %q", itemPath)
+			var dest string
+
+			dest, err = os.Readlink(path)
+			if err != nil {
+				return err
+			}
+
+			if strings.IndexByte(dest, '/') != -1 || dest == "." || dest == ".." {
```

Comment:
a symlink like this could potentially skip this test `ln -s ./../notes.txt foo`?

Reply:
nope, as we don't allow slashes for now, so symlink could only be a filename like `a.so` -> `a.so`